api_proxy.yaml

```diff
@@ -393,6 +416,22 @@ definitions:
       - name
       - preferred_email
       - version
+  PublicSession:
+    type: object
+    title: User session data
+    description: Describe the current session of an authenticated user.
+    properties:
+      expired:
+        type: boolean
+      expireAt:
+        type: number
```

Comment:
this should be an ISO timestamp

Reply:
ref. https://github.com/teamdigitale/digital-citizenship-functions/blob/master/api/definitions.yaml#L428

Reply:
@fpersico do you see any problem in switching from a unix epoch timestamp to a ISO timestamp in the app?

Reply:
btw swagger specs supports only date-time = ISO time

Reply:
isostring is good